tests/test_D1_model_components.py

```diff
@@ -675,6 +678,15 @@ def test_storage_optimize(self):
             == dict_asset[OUTPUT_POWER][C_RATE][VALUE]
         )
 
+        assert (
+            self.model.entities[-1].fixed_losses_relative.default
+            == dict_asset[STORAGE_CAPACITY][THERM_LOSSES_REL][VALUE]
+        )
+        assert (
+            self.model.entities[-1].fixed_losses_absolute.default
+            == dict_asset[STORAGE_CAPACITY][THERM_LOSSES_ABS][VALUE]
+        )
+
     def test_storage_fix(self):
```

Comment:
For a better overview could you put `test_storage_fix()` either to the top or bottom of this part, so that all test functions for one function come one after another?

Reply:
Good point! Done with commit https://github.com/rl-institut/multi-vector-simulator/pull/718/commits/6005fc7a88893ea15c0bccb70d8c5bd596c8fe5c.